app-config.production.yaml

```diff
@@ -0,0 +1,34 @@
+app:
```

Comment:
It looks like we're using `app-config.yaml` as our production config, should this file be deleted?  Or should we update it so that this is used in production and the other file is setup to be able to do local dev for this repo?

Reply:
Given this, I think my vote would be to delete the prod file, what do you think.

> Configuration is stored in YAML files where the defaults are app-config.yaml and app-config.local.yaml for local overrides. Other sets of files can by loaded by passing --config <path> flags. The configuration files themselves contain plain YAML, but with support for loading in data and secrets from various sources using for example $env and $file keys.